lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -68,24 +68,32 @@ Function Get-DomainMembershipMatch {
     }
 }
 
-Function Create-Credential {
-    Param(
-        [string] $cred_user,
-        [string] $cred_pass
-    )
-
-    $cred = New-Object System.Management.Automation.PSCredential($cred_user, $($cred_pass | ConvertTo-SecureString -AsPlainText -Force))
-
-    return $cred
+<#
+    .SYNOPSIS
+    Save the username and the encrypted password as PSCredential object
+    .Parameter username
+    Username of the user which will join or unjoin the computer object
+    .Parameter passwordEncrypted
+    Encrypted password of the user which will be used for join or unjoin the computer object
+#>
+function Save-Credential {
```

Comment:
We aren't actually "Saving" a credential, the docs for `Save` is

> Preserves data to avoid loss.

https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands

Technically `New` or `ConvertTo-Credentials` would be a better fit here. I also don't see the main point of now accepting only a SecureString. It seems like the main benefit before was that it was a helper function to easily convert the string inputs from Ansible to a PSCredential whereas now we still need to do some work outside to convert the pass to a SecureString.

In reality I don't see the need for this cmdlet as it's only being called once. If you are going to  refactor this part I would remove it altogether and just create the credentials where this is being called.